src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/raw/RabbitRawBatchSender.java

```diff
@@ -58,7 +63,16 @@ protected int extractCountFrom(RawMessageBatch message) {
     }
 
     @Override
-    protected String toShortDebugString(RawMessageBatch value) {
+    protected String toShortTraceString(RawMessageBatch value) {
         return MessageUtils.toJson(value);
     }
+
+    @Override
+    protected String toShortDebugString(RawMessageBatch value) {
+        String[] sessionAliasAndDirection = getSessionAliasAndDirection(value.getMessages(0).getMetadata().getId());
+        return String.format("RawMessageBatch: session_alias = %s, direction = %s, sequences = %s",
+                sessionAliasAndDirection[0],
+                sessionAliasAndDirection[1],
+                value.getMessagesList().stream().map(message -> Long.toString(message.getMetadata().getId().getSequence())).collect(Collectors.joining(", ")));
```

Comment:
Please extract to a Utility class to use the same version of code in a publisher and a subscriber.